doc/api/modules.md

```diff
@@ -175,15 +175,15 @@ LOAD_AS_DIRECTORY(X)
 2. LOAD_INDEX(X)
 
 LOAD_NODE_MODULES(X, START)
-1. let DIRS=NODE_MODULES_PATHS(START)
+1. let DIRS = NODE_MODULES_PATHS(START)
 2. for each DIR in DIRS:
    a. LOAD_AS_FILE(DIR/X)
    b. LOAD_AS_DIRECTORY(DIR/X)
 
 NODE_MODULES_PATHS(START)
 1. let PARTS = path split(START)
 2. let I = count of PARTS - 1
-3. let DIRS = []
+3. let DIRS = [GLOBAL_FOLDERS](#modules_loading_from_the_global_folders)
```

Comment:
Will this link render on the website? On github it's not.

Reply:
[It seems it will)](https://nodejs.org/download/nightly/v11.0.0-nightly201805075efbe4c1e8/docs/api/modules.html#modules_loading_from_the_global_folders)

Reply:
![screenshot from 2018-05-07 12-31-27](https://user-images.githubusercontent.com/1493671/39697825-93233086-51f2-11e8-8503-058378ef81a9.png)

It seems not up-to-date, or is it just me?

Reply:
This is just heading in the last nightly doc to illustrate link validity, the content will be changed by this PR.

Reply:
GitHub pages and rendered pages on the site use different header ids.

Reply:
Sorry, I misunderstood the note.